packages/lib-classifier/src/plugins/drawingTools/components/Mark/Mark.js

```diff
@@ -39,11 +40,16 @@ const Mark = forwardRef(function Mark ({
   }
   const focusColor = theme.global.colors[theme.global.colors.focus]
 
-  React.useEffect(() => {
-    if (mark.finished && !mark.subTaskVisibility) {
-      onFinish()
+  function openSubTaskPopup() {
+    if (mark.finished &&
+      !mark.subTaskVisibility &&
+      mark.tasks.length > 0
+    ) {
+      mark.setSubTaskVisibility(true, markRoot.current)
```

Comment:
I _think_ we could pass the mark's bounding client rectangle here, as the second arg, instead of a ref. I'm not sure.

Reply:
Seems to work, so I've committed that change.